tutorials/templates/add.html

```diff
@@ -36,7 +36,14 @@ <h2 class="site-header">
 {% block after-body-bottom-js %}
     <script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
     <script>
-        var easyMDE = new EasyMDE({element: document.getElementById('id_markdown'), nativeSpellcheck: false, forceSync: true});
+        var easyMDE = new EasyMDE({element: document.getElementById('id_markdown'),
+                                    nativeSpellcheck: false,
+                                    forceSync: true,
+                                    showIcons: ['upload-image'],
+                                    uploadImage: true,
+                                    imageMaxSize: '100px',
+                                    imageUploadEndpoint: 'http://openenergy-platform.org/api/v0/image/upload/',
```

Comment:
The URL should not be hard-coded
```suggestion
                                    imageUploadEndpoint: '/api/v0/image/upload/',
```